cmds/core/ls/ls.go

```diff
@@ -34,6 +34,7 @@ var (
 	human     = flag.BoolP("human-readable", "h", false, "human readable sizes")
 	directory = flag.BoolP("directory", "d", false, "list directories but not their contents")
 	long      = flag.BoolP("long", "l", false, "long form")
+	final     = flag.BoolP("print-last", "p", false, "Print only the final path element of each file name")
```

Comment:
can you put this flag in a file called 
ls_plan9.go
so it won't appear when unix folks do an ls -h?